src/Illuminate/Queue/ListenerOptions.php

```diff
@@ -21,10 +28,12 @@ class ListenerOptions extends WorkerOptions
      * @param  int  $sleep
      * @param  int  $maxTries
      * @param  bool  $force
+     * @param  int  $verbosity
```

Comment:
@lucasmichot used to be an integer :)) thanks for feedback, everything is fixed except strict type for exception in the constructor (I assumed if another class does it a certain way, I should do the same way for consistency?)